arbeitszeit_flask/templates/base_company.html

```diff
@@ -63,46 +63,46 @@
 
           <div class="navbar-item has-dropdown is-hoverable">
             <a class="navbar-link">
-              Konten und Käufe
+              {{ gettext("Accounts and purchases") }}
             </a>
 
             <div class="navbar-dropdown">
               <a href="{{ url_for('main_company.my_accounts') }}" class="navbar-item">
-                Konten
+                {{ gettext("Accounts") }}
               </a>
               <a href="{{ url_for('main_company.transfer_to_worker') }}" class="navbar-item">
-                Bezahlen Mitarbeiter
+                {{ gettext("Pay workers") }}
               </a>
               <a href="{{ url_for('main_company.transfer_to_company') }}" class="navbar-item">
-                Bezahlen Produktionsmittel
+                {{ gettext("Pay means") }}
```

Comment:
For "Produktionsmittel" maybe "means of prodcution" or "resources" would be better

Reply:
> @knit-bee I implemented the suggestions.

Don't want to be petty, but would "Pay means of production" be ok as well?